packages/ag-charts-community/src/chart/data/processors.ts

```diff
@@ -215,20 +215,20 @@ function normaliseFindExtent(mode: 'sum' | 'range', columns: any[][], valueIndex
     for (const valueIdx of valueIndexes) {
         const column = columns[valueIdx];
         const value: null | number | (null | number)[] = column[datumIndex];
-        if (value == null) continue;
-        // Note - Array.isArray(new Float64Array) is false, and this type is used for stack accumulators
-        const valueExtent = typeof value === 'number' ? value : Math.max(...value.map((v) => v ?? 0));
-        const valIdx = valueExtent < 0 ? 0 : 1;
+        if (value == null || typeof value !== 'number') continue;
```

Comment:
The array here looked to just be an accumulator, the actual values are the non-array ones so I believe we can just skip those?